changelogs/unreleased/gh-9849-grant-revoke-user-right-after-upgrade.md

```diff
@@ -1,2 +1,6 @@
 ## feature/box
 * Added `on_schema_upgrade_end` trigger (gh-9849).
+
+## bugfix/box
+* User rights are now granted/revoked after upgrading automatically without
+  restarting (gh-9849).
```

Comment:
Nit: The commit message has no 'fixes/closes #xxxx'.

Reply:
Thank you. Fixed it.